airflow/contrib/hooks/aws_athena_hook.py

```diff
@@ -148,3 +156,105 @@ def stop_query(self, query_execution_id):
         :return: dict
         """
         return self.conn.stop_query_execution(QueryExecutionId=query_execution_id)
+
+
+class AWSAthenaHelpers(AWSAthenaHook):
```

Comment:
This class name doesn't follow any of Airflow's conventions.

This feels like an operator, not a hook class.

Reply:
Because original Athena didn't support INSERT INTO TABLE clause, so I'm not what kind of this function belong to.